test_util/test_installer_ccm.py

```diff
@@ -257,13 +257,16 @@ def prep_hosts(ssh_runner, registry):
     check_results(run_loop(ssh_runner, host_prep_chain))
 
 
-def make_vpc(use_bare_os=False):
+def make_vpc(count, use_bare_os=False):
     """uses CCM to provision a test VPC of minimal size (3).
 
     Args:
+        count: number of instances to spin up
         use_bare_os: if True, vanilla AMI is used. If False, custom AMI is used
             with much faster prereq satisfaction time
     """
+    assert count >= 4, ('Cluster must be at least 4 hosts: '
```

Comment:
In that case, the code will no longer be cleanup, so I'll take it out of here for now and put it in another PR so we can get the other two basic commits in